### PR TITLE
Add description field to ServiceClass

### LINKED ISSUE #654: ServiceClass description not being passed through
According to the OSB api spec description goes directly on the service object and not the metadata blob.  So brokers properly returning description on their services are not getting that description coming back through the svc cat api on ServiceClass.

### PR DESCRIPTION
**Pull request by jpeeler**
Fixes #654

### CODE REVIEW

**Comment by pmorie** [comment]
Have you tested this with the ups broker?

File: pkg/apis/servicecatalog/v1alpha1/types.go [hunk 1]
```diff
@@ -138,15 +139,14 @@ type ServicePlan struct {
...
-	Description *string `json:"description,omitempty"`
-
 	// OSB-specific
 	// OSBGUID is the identity of this object for use with the OSB API.
 	// Immutable.
 	OSBGUID string `json:"osbGuid"`
 
 	// OSB-specific
 	OSBFree     bool                  `json:"osbFree"`
+	Description *string               `json:"description,omitempty"`
```
Comment (pmorie) [inline pkg/apis/servicecatalog/v1alpha1/types.go]: Add a godoc comment for this field

File: pkg/apis/servicecatalog/types.go [hunk 2]
```diff
@@ -130,6 +130,7 @@ type ServiceClass struct {
 	OSBDashboardOAuth2ClientID *string
 	OSBDashboardSecret         *string
 	OSBDashboardRedirectURI    *string
+	Description                *string
```
Comment (pmorie) [inline pkg/apis/servicecatalog/types.go]: Add a godoc comment for this field

**Comment by jpeeler** [comment]
Test results:

apiVersion: servicecatalog.k8s.io/v1alpha1
bindable: false
brokerName: ups-broker
description: User Provided Service
kind: ServiceClass
metadata:
  creationTimestamp: 2017-04-07T18:30:36Z
  name: user-provided-service
  resourceVersion: "5"
  selfLink: /apis/servicecatalog.k8s.io/v1alpha1/serviceclassesuser-provided-service
  uid: 4ad6af2f-1bc0-11e7-9090-0242ac110004
osbGuid: 4f6e6cf6-ffdd-425f-a2c7-3c9258ad2468
osbMetadata: null
planUpdatable: false
plans:
- name: default
  osbFree: true
  osbGuid: 86064792-7ea2-467b-af93-ac9694d96d52
  osbMetadata: null

**Comment by pmorie** [comment]
Excellent.

**Comment by simonleung8** [comment]
LGTM

**Pull request merged**